recipe/meta.yaml

```diff
@@ -9,37 +9,30 @@ source:
   sha256: e9ac89baab59dbb585c38f8b8449627d53b57ae79100d8bbca836907c0b6c27a
   patches:
     - xmltester.cpp.patch  # [win]
-    - DirectedEdgeStar_vc9.patch  # [win and py27]
-    - GEOSGeom_createCollection_vc9.patch  # [win and py27]
+    - DirectedEdgeStar_vc9.patch  # [win and vc==9]
+    - GEOSGeom_createCollection_vc9.patch  # [win and vc==9]
```

Comment:
These last 2 patches aren't being applied (see https://ci.appveyor.com/project/conda-forge/geos-feedstock/build/1.0.79/job/jx8gp9ta90llknse?fullLog=true#L195). I think the build would work on Windows/vc9 if they were...

Reply:
Good eye, Sam.  I'll look into why.  They absolutely should be.  Could be a problem with the "used variable" detection stuff.